schedule/yast/autoyast_create_hdd_textmode_raid_encrypted_home.yaml

```diff
@@ -0,0 +1,14 @@
+name: autoyast_create_hdd_gnome_raid_encrypted_home
```

Comment:
also in name field you need an update to 'textmode'

Reply:
updated.